src/animator.js

```diff
@@ -51,40 +51,46 @@ export class VelocityAnimator {
    * @param element Element or array of elements to animate
    * @param nameOrProps Element properties to animate
    * @param options Animation options
-   * @return resolved when animation is complete
+   * @param silent Disable animation events
+   * @return resolves to Array (element was an HTMLElement) or NodeList (element was a string or NodeList) when complete
    */
-  animate(element:any, nameOrProps:any, options?:any, silent?:boolean): Promise<VelocityAnimator> {
+  animate(element: any, nameOrProps: any, options?: Object, silent?: boolean): Promise<any> {
+    // validate input
+    if (!element) return Promise.reject(new Error('first argument (element) must be defined'));
```

Comment:
Hmm, doesn't seem like a breaking change, for it would fail with the error later (due to the fact that `velocity()` does not return a Promise), and be less descriptive.